app/controllers/ManageGroupController.scala

```diff
@@ -54,9 +54,7 @@ class ManageGroupController @Inject()(
      confirm_delete_group: confirm_delete_group,
      delete_group_complete: delete_group_complete,
      review_clients_to_add: review_clients_to_add,
-     client_group_list: client_group_list,
      clients_update_complete: clients_update_complete,
```

Comment:
I would expect `clients_update_complete` and `review_clients_to_add` to also move across - are they still needed in this controller?

`showGroupClientsUpdatedConfirmation` looks duplicated

Reply:
currently yes.